src/plugins/intel_cpu/src/nodes/tensoriterator.cpp

```diff
@@ -270,19 +299,9 @@ std::shared_ptr<dnnl::memory> DynamicBuffer::create_buffer(const dnnl::engine& e
     const auto old_desc = mem_holder_buffer->get_desc();
     auto dims = old_desc.dims();
 
-    if (from->getStaticDims()[axis] != abs_stride)
-        IE_THROW() << "TensorIterator (Loop) has incorrect output shape[axis] after iteration for concatenation. " << abs_stride <<
-        " is expected, but actual: " << from->getStaticDims()[axis];
-
-    dims[axis] += abs_stride;
+    dims[axis] += abs_stride * 1;  // TODO: the increasing amount
```

Comment:
Do you plan to implement this within this PR?

Reply:
updated. with growth_factor=2.0